packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1952,7 +1976,12 @@ export default class Meeting extends StatelessWebexPlugin {
 
         .then(() => {
           if (mediaSettings && mediaSettings.sendShare && localShare) {
-            return this.share();
+            if (this.state === MEETING_STATE.STATES.JOINED) {
```

Comment:
nice

Reply:
Should we print an error if the meeting isn't joined?

Reply:
We dont have to because its a valid case as server delays the join sometimes